docs/devel/pull-requests.md

```diff
@@ -73,16 +73,12 @@ The following will save time for both you and your reviewer:
 
 ## Release Notes
 
-All pull requests are initiated with a `needs-release-note` label.
-There are many `release-note-*` label options, including `release-note-none`.
-If your PR does not require any visibility at release time, you may use a
-`release-note-none` label.  Otherwise, please choose a `release-note-*` label
-that fits your PR.
-
-Additionally, `release-note-none` is not allowed on PRs on release branches.
-
-Finally, ensure your PR title is the release note you want published at relase
-time.
+1. Your PR title is the **release note** you want published at release time.
```

Comment:
Title is insufficient and in many cases inappropriate. 

Compare:
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md#other-notable-improvements

and:
https://github.com/kubernetes/kubernetes/releases/tag/v1.1.2

I want the former, not the latter.